src/tox/config/loader/str_convert.py

```diff
@@ -63,6 +63,11 @@ def to_command(value: str) -> Command:
                 pos = splitter.instream.tell()
         except ValueError:
             args.append(value[pos:])
+        if len(args) == 0:
+            raise TypeError(
+                f"Attempting to parse {value!r} into a command failed: no command was found. "
+                "Check your tox configuration for environments with missing command values."
```

Comment:
I think `f"Attempting to parse {value!r} into a command failed"` suffices as message here.